app/portainer/components/be-feature-indicator/be-feature-indicator.css

```diff
@@ -7,7 +7,8 @@
   pointer-events: all;
 }
 
-.be-indicator .icon {
+.be-indicator .icon,
+.be-indicator .fas {
```

Comment:
and where is this used?

Reply:
.icon was working, but I can't see CSS class icon anymore in the <i> element, so I added the .fas which is in the <i> element/

Reply:
I didn't remove .icon just in case it is still in use in other places.

Reply:
I suggest adding `icon` class to the icon. `fas` won't work if we change the icon.
also, since we have `transclude` inside the component, maybe name the class `be-indicator-icon`, so it won't overwrite other icons styles